client/blocks/login/login-form.jsx

```diff
@@ -249,11 +257,12 @@ export class LoginForm extends Component {
 					</div>
 				</Card>
 				{ config.isEnabled( 'signup/social' ) && (
-					<Card className="login__form-social">
-						<SocialLoginForm
-							onSuccess={ this.props.onSuccess }
-							linkSocialUser={ this.linkSocialUser }
-							linkingSocialService={ this.state.linkingSocialService } />
+					<Card>
+						<div className="login__form-social">
```

Comment:
Just out of curiosity, what's the rationale for nesting an additional div here? I tried messing around with the login styling and couldn't really tell the difference.